docs/hypernode-platform/mysql/how-to-use-mysql-on-hypernode.md

```diff
@@ -196,6 +196,31 @@ TABLE="core_url_rewrite"
 mysql "$DATABASE" -e "TRUNCATE TABLE $TABLE"
 ```
 
+## Changing the Authentication Plugin for a MySQL User
+
+Some MySQL features and upgrades require that users use a supported authentication plugin.
```

Comment:
We already do this for the `app` user right? Maybe good to highlight that it is only required for custom new users, I don't think I've ever had any experience with our users setting up new sql users however.